test/tap/index.js

```diff
@@ -0,0 +1,64 @@
+const test = require('tap').test
```

Comment:
Was think about having a `test/lib` dir for test helpers, but it's YAGNI for now and I'll cross that bridge later should it surface.